src/environments/environment.ts

```diff
@@ -26,11 +26,11 @@ export const environment = {
     serverUrl: 'https://accounts.test.oneacrefund.org',
     realm: 'OneAcreFund',
     client_id: 'fineract',
-    redirectUri: window['env']['homeURL'] || 'http://localhost:4200/home'
+    tokenUrl: `https://loans.test.oneacrefund.org/auth/realms/OneAcreFund/protocol/openid-connect/token`,
+    redirectUri: window['env']['homeURL'] || 'http://localhost:4200/#products/loan-products/create'
```

Comment:
@amydoxym any reason for using the redirect uri to point to loan products if the homeUrl is not availabe?

Reply:
nice catch! Thanks..Corrected it, it was just a shortcut during development